src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1117,10 +1132,12 @@ public void onWorldTick(@NotNull final TickEvent.WorldTickEvent event)
                 }
             }
 
-            if (shallUpdate(world, TICKS_SECOND) && event.world.getDifficulty() != EnumDifficulty.PEACEFUL
+            if (shallUpdate(world, TICKS_SECOND)
+                    && event.world.getDifficulty() != EnumDifficulty.PEACEFUL
                     && Configurations.gameplay.doBarbariansSpawn
+                    && canHaveBarbEvents
                     && !world.getMinecraftServer().getPlayerList().getPlayers()
-                    .stream().filter(permissions::isSubscriber).collect(Collectors.toList()).isEmpty()
+                        .stream().filter(permissions::isSubscriber).collect(Collectors.toList()).isEmpty()
                     && MobEventsUtils.isItTimeToRaid(event.world, this))
             {
                 MobEventsUtils.barbarianEvent(event.world, this);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS106)